### PR TITLE
Fixing git

### PR DESCRIPTION
**Pull request by smkniazi**

### CODE REVIEW

File: main.go [hunk 1]
```diff
@@ -73,7 +73,7 @@ func main() {
 	}
 
 	// Creating the virtual file system
-	fileSystem, err := NewFileSystem(ftHdfsAccessor, mntSrcDir, allowedPrefixes, *expandZips, *readOnly, retryPolicy, WallClock{})
+	fileSystem, err := NewFileSystem(ftHdfsAccessor, mntSrcDir, allowedPrefixes /**expandZips*/, false, *readOnly, retryPolicy, WallClock{})
```
Comment (maismail) [inline main.go:76]: will it be fixed later? if so add a fixme and create a ticket for it

File: HopsFileHandle.go [hunk 2]
```diff
@@ -23,6 +24,7 @@ type FileHandle struct {
 	fileFlags         fuse.OpenFlags // flags used to creat the file
 	tatalBytesRead    int64
 	totalBytesWritten int64
+	FHID              int64 // file handle id. for debugging only
```
Comment (maismail) [inline HopsFileHandle.go:27]: do you need it to be global? maybe `fHID` instead?

Comment (smkniazi) [reply HopsFileHandle.go]: fxd

File: main.go (hunk 1 above)
Comment (smkniazi) [reply main.go:76]: fxd

**Review by maismail — APPROVED** [review]

**Pull request merged**